src/app/core/services/common-api.service.ts

```diff
@@ -35,8 +35,8 @@ export class CommonApiService {
 
   constructor(protected http: Http) { }
 
-  fetchUrl(url: string): Promise<Object> {
-    return this.http.get(url)
+  fetchUrl(url: string, headers?: Headers): Promise<Object> {
```

Comment:
let's make this exactly same signature of as the `http.get`, so second param would be `options` I think.

Reply:
sorry if I wasn't clear.